mbuild/conversion.py

```diff
@@ -33,6 +34,7 @@ def load(filename_or_object,
          infer_hierarchy=True,
          backend=None,
          ignore_box_warn=False,
+         smiles_seed=0,
```

Comment:
```suggestion
         smiles_seed=-1,
```

Reply:
wouldn't we want a seed to be set by default?

Reply:
Yeah that was my thought too. Probably a bit better to have a set specified here by default. then provide the user information about `-1` as an option to ignore setting a PRNG seed.